### PR TITLE
Adding Create Route Skeleton

### PR DESCRIPTION
**Pull request by ahadinger**
Added create route button and started setting up the form.
Also fixed issue with the stops popup not working after moving to using the passio api.

### CODE REVIEW

File: src/CreateRoute.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { CreateRouteForm } from "./util/CreateRouteForm";
+
+function CreateRouteButton() {
+    const [showCRPopup,setCRPopup] =  useState(false)
+    var right = 0 + 'px';
+    var bottom = 0 + 'px';
+    var padding = 10 + 'px';
+    return (
+      <div style={{padding, right, bottom,position:'absolute', zIndex: 1, display: 'flex'}}>
+        {showCRPopup === false ? (
```
Comment (kalmasri123) [inline src/CreateRoute.js:15]: checking equality of a boolean is redundant

File: src/routes.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+[
+    {
+        "id": "3201",
+        "name": "Silver",
+        "routes": [
```
Comment (kalmasri123) [inline src/routes.json:5]: this is a nitpick but shouldnt this be stops?

File: src/types/Stop.ts [hunk 3]
```diff
@@ -10,5 +10,7 @@
     routeGroupId:   number;
     location: { lat: number; lng: number };
     position:string;
+    route_list: [];
```
Comment (kalmasri123) [inline src/types/Stop.ts:13]: if its possisble can this be camelcase in the future?

**Review by kalmasri123 — COMMENTED** [review]
poggers

File: src/routes.json (hunk 2 above)
Comment (ahadinger) [reply src/routes.json:5]: it should, oopsies

File: src/types/Stop.ts (hunk 3 above)
Comment (ahadinger) [reply src/types/Stop.ts:13]: I can do that

Comment (ahadinger) [reply src/types/Stop.ts]: changed to camelcase

File: src/routes.json (hunk 2 above)
Comment (ahadinger) [reply src/routes.json]: fixed this, doh

**Comment by ahadinger** [comment]
Merging so we can work from this

**Pull request merged**